docs/utils/mdHelper.js

```diff
@@ -0,0 +1,36 @@
+// @flow strict
+import path from 'path';
+import fs from 'fs';
+import matter from 'gray-matter';
+
+export function getDocByRoute(route: string): {|
+  content?: string,
+  meta: { [key: string]: string },
+  route: string,
+  isMDX: boolean,
+|} {
+  const docsDirectory = path.join(process.cwd(), 'markdown');
+  const fullPath = path.join(docsDirectory, `${route}.md`);
+
+  try {
+    const fileContents = fs.readFileSync(fullPath, 'utf8');
```

Comment:
Do the file/dir reads in this file need to be sync?

Reply:
Good catch, will update them to be async